css/style.css

```
@@ -1,6 +1,20 @@
 * {
   box-sizing: border-box;
   -webkit-tap-highlight-color: transparent;
+  scroll-behavior: smooth;
+}
+
+/* handle header offset with scroll-margin */
+h1, h2, h3, h4, h5, h6,
+[id] {
+  scroll-margin-top: 120px;

```

```suggestion
  scroll-margin-top: 60px;
```
see comment https://github.com/expressjs/expressjs.com/pull/1965/files#r2206464995